application/lib/utils/route_constants.dart

```diff
@@ -2,4 +2,8 @@ const String startRoute = '/start';
 const String signUpRoute = '/signup';
 const String logInRoute = '/login';
 const String homeRoute = '/home';
-const String organizerHomeRoute = '/organizer_home';
+const String profileRoute = '/profile';
```

Comment:
Itt véletlen bekerültek olyan route-ok is, amik még nincsenek is használva. Azokat kérlek szedjük ki.